packages/backend/src/api/routes/auth.ts

```diff
@@ -0,0 +1,68 @@
+import type { Services } from '@services';
+import { type Request, Router } from 'express';
+import type { ApiRoutes, UserSignInDto, UserSignUpDto } from '@vse-bude/shared';
+import { AuthApiRoutes } from '@vse-bude/shared';
+import { wrap } from '@helpers';
+import { apiPath } from '../../helpers/api';
+import { authMiddleware } from '../../auth/middlewares/auth.middleware';
+import type { SignOutDto } from '../../common/types/auth/sign-out.dto';
+import type { UpdateRefreshTokenDto } from '../../common/types/refresh-token/update.refresh-token.dto';
+
+export const initAuthRoutes = (
+  { authService }: Services,
+  path: ApiRoutes,
+): Router => {
+  const router = Router();
+
+  router.post(
+    apiPath(path, AuthApiRoutes.SIGN_IN),
+    wrap(async (req: Request) => {
```

Comment:
here you can use `wrap<Empty, TokensDto, UserSignInDto>`, and then just use 

`return authService.signIn(req.body);`